packages/qwik/src/server/v2-ssr-container.ts

```diff
@@ -970,29 +976,29 @@ class SSRContainer extends _SharedContainer implements ISSRContainer {
   }
 
   /**
-   * @param tag
+   * @param elementName
    * @param depthFirstElementIdx
    * @param isElement
    */
```

Comment:
let's align with current function's arguments..

why do we even need this js docs block? seems to not add much.

Reply:
yeah I will remove it